backend/app_fastapi_v2.py

```
@@ -24,7 +24,7 @@
     create_review
 )
 from backend.models.user import User, Review
-from backend.tests.test_data.test_review_data import TEST_DATA_REVIEW
+from tests_backend.test_data.test_review_data import TEST_DATA_REVIEW
```

Ruff

`tests_backend.test_data.test_review_data.TEST_DATA_REVIEW` imported but unused

Remove unused import: `tests_backend.test_data.test_review_data.TEST_DATA_REVIEW`

(F401)